pkg/cidata/cidata.TEMPLATE.d/boot/20-rootless-base.sh

```diff
@@ -63,7 +63,9 @@ fi
 
 # Set up subuid
 for f in /etc/subuid /etc/subgid; do
-	grep -qw "${LIMA_CIDATA_USER}" $f || echo "${LIMA_CIDATA_USER}:100000:65536" >>$f
+	# systemd-homed expects the subuid range to be within 524288-1878982656 (0x80000-0x6fff0000).
+	# See userdbctl.
+	grep -qw "${LIMA_CIDATA_USER}" $f || echo "${LIMA_CIDATA_USER}:524288:1073741824" >>$f
```

Comment:
The comment doesn't make it clear to me why you picked 1073741824 (`0x40000000`) as the upper bound.

Reply:
1073741825-1878982656 is left blank for additional users

Reply:
Ok, so it is kind of arbitrary? Don't know if you want to add a comment about it or not.

Reply:
Added